docs/funding.md

```diff
@@ -0,0 +1,53 @@
+# Announcing HTTPX 1.0
+
+HTTPX is a fully featured HTTP client for Python, which offers both HTTP/1.1 and HTTP/2 support. Learning from all of the functionality and ease of use that the fantastic `requests` library provides, while bringing additional features and keeping pace with HTTP as it evolves.
+
+* An API that is broadly compatible with `requests`.
+* Fully supports all the features that the `requests` package provides.
+* Both HTTP/1.1 and HTTP/2 support.
+* Strict request timeouts enabled by default.
+* Support for the asyncio and trio concurrency environments.
+* A fully type annotated interface, with 100% code coverage.
+* Development & maintenance that's backed by a sustainable and transparent business model.
+
+HTTPX is the **only** Python HTTP client with **HTTP/1.1 and HTTP/2** support, as well as being the only client providing both **sync and async** support. We'd like to continue pushing the project forward, and we're asking for sponsors to help us in building the foundation of modern HTTP stack for Python.
+
+We've spent a long time finessing the API for the project, and we're now happy to release a 1.0 version, and provide a standard SEMVER policy on future API updates.
```

Comment:
I think you should flesh out the stability policy per https://docs.twistedmatrix.com/en/latest/core/development/policy/compatibility-policy.html

and include a separate policy for type hints, and behaviour under adverse conditions such as cancellation